jobs/V3/email_alert_search_sample.rb

```diff
@@ -0,0 +1,75 @@
+# Copyright 2018 Google, Inc
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+def job_discovery_email_alert_search google_cloud_project_id:, company_name:
+  # [START job_discovery_email_alert_search]
+  # google_cloud_project_id = "Id of the project"
```

Comment:
@baochanzheng I have one request that I missed in the prior PRs. Could you update the variable `google_cloud_project_id` to `project_id` across samples? It will help reduce line length.